transportable-udfs-trino-plugin/src/test/java/com/linkedin/transport/trino/plugin/TestTransportUdf.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino.plugin;
+
+import com.linkedin.transport.trino.TransportPlugin;
+import io.trino.server.testing.TestingTrinoServer;
+import java.sql.Statement;
+import org.testng.annotations.BeforeClass;
+
+
+public class TestTransportUdf {
```

Comment:
Where is it used? Can we add unit tests with it?

Reply:
+1 it seems we don't actually have any tests here currently ... ? Especially with the custom classloader there is a lot of logic in this module and currently we don't have any tests, that's no good!

Reply:
This class is for experiments and not needed. I have removed it. The unit test is added in the class named `TransportPluginTest`